daffodil-test/src/test/resources/org/apache/daffodil/section05/simple_types/SimpleTypes.tdml

```diff
@@ -3719,6 +3722,17 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
+  <tdml:parserTestCase name="float_text_delim" root="floatTextDelim"
+                       model="SimpleTypes-Embedded.dfdl.xsd" description="Section 5 Simple type-float - DFDL-5-008R">
+
+    <tdml:document><![CDATA[3.4028235E38]]></tdml:document>
```

Comment:
The document does not end with the terminator (%SP;) expected by the floatTextDelim type.  Can you comment on what the implications are?

Reply:
Fields with `lengthKind="delimited"` can be delimited by more than just separators, including EOF. From the spec:

> On parsing, the length of an element with dfdl:lengthKind 'delimited' is determined by scanning the data stream for the delimiter.
> 
> The data stream is scanned for any of
> ·         the element's terminator (if specified)
> ·         an enclosing construct's separator or terminator
> ·         the end of an enclosing element designated by its known length
> ·         the end of the data stream

So the dfdl:separator="%SP;" isn't really needed in this case, but it doesn't hurt.